expression/column_test.go

```diff
@@ -172,7 +172,7 @@ func (s *testEvaluatorSuite) TestColHybird(c *C) {
 		c.Assert(err, IsNil)
 		input.AppendBytes(0, num)
 	}
-	result, err := newBuffer(types.ETInt, 1024)
+	result, err := newBuffer(types.ETString, 1024)
```

Comment:
Could you add an annotation to explain why use `VecEvalInt` but type is `ETString` here?

Reply:
Maybe we should keep this regular test and add a new specific case.

Reply:
The case can be triggered if the result `chunk.Chunk` has a different field type with `Int`.

Reply:
I get it, but it is not intuitive for the first-time code reader.

Reply:
Comments addressed